pkg/server/helpers_selinux_test.go

```diff
@@ -31,27 +32,46 @@ func TestInitSelinuxOpts(t *testing.T) {
 		return
 	}
 
+	var pLabelNolevel, mLabelNolevel string
+	pLabel, mLabel := selinux.ContainerLabels()
+	if pLabel != "" && mLabel != "" {
+		pcon := selinux.NewContext(pLabel)
+		mcon := selinux.NewContext(mLabel)
+		pLabelNolevel = fmt.Sprintf("%s:%s:%s", pcon["user"], pcon["role"], pcon["type"])
+		mLabelNolevel = fmt.Sprintf("%s:%s:%s", mcon["user"], mcon["role"], mcon["type"])
+	}
+
 	for desc, test := range map[string]struct {
 		selinuxOpt   *runtime.SELinuxOption
 		processLabel string
 		mountLabels  []string
 	}{
-		"testNullValue": {
+		"Should return empty strings for processLabel and mountLabel when selinuxOpt is nil": {
 			selinuxOpt:   nil,
 			processLabel: "",
 			mountLabels:  []string{"", ""},
 		},
-		"testEmptyString": {
+		"Should be resolved correctly when selinuxOpt is not nil but has not been initialized": {
 			selinuxOpt: &runtime.SELinuxOption{
 				User:  "",
 				Role:  "",
 				Type:  "",
 				Level: "",
 			},
-			processLabel: ":::",
-			mountLabels:  []string{":object_r:container_file_t:", ":object_r:svirt_sandbox_file_t:"},
+			processLabel: pLabelNolevel,
+			mountLabels:  []string{mLabelNolevel, ""},
+		},
+		"Should be resolved correctly when selinuxOpt has been initialized partially": {
```

Comment:
So these should be ignored if it's incomplete?

Reply:
I think this should depend on the implementation of k8s. 

These should be ignored if K8s don't allow to set incomplete selinux options.

Reply:
So `InitLabels` will ignore them if they are incomplete?

Reply:
No, `InitLabels` have no error returned when the selinux option is incomplete.

Reply:
But must be in this form: `user: , role: , type: , level: ,`
For example, `user: , role:user_r, type: , level:s0:c1,c2,`

Reply:
Refer to the [test case](
 https://github.com/opencontainers/selinux/blob/4a2974bf1ee960774ffd517717f1f45325af0206/go-selinux/label/label_selinux_test.go#L49-L54) for `InitLabels`, maybe we should ignore these?

But the options is compliant here.